ssl-config/src/main/java/com/palantir/remoting/ssl/SslSocketFactories.java

```diff
@@ -59,12 +60,12 @@ public static SSLContext createSslContext(SslConfiguration config) {
         }
     }
 
-    private static TrustManagerFactory createTrustManagerFactory(TrustStoreConfiguration trustStoreConfig) {
+    private static TrustManagerFactory createTrustManagerFactory(SslConfiguration sslConfig) {
```

Comment:
the args to this should probably explicitly be `Path trustStorePath`, `Optional<String> trustStorePassword`, `String trustStoreType`